README.md

```diff
@@ -1,8 +1,9 @@
 [![deploy to gh-pages](https://github.com/Minimal-APIs/minimal-apis.github.io/actions/workflows/deploy.yml/badge.svg)](https://github.com/Minimal-APIs/minimal-apis.github.io/actions/workflows/deploy.yml)
+
 # minimal.github.io
-tutorials and samples for minimal apis 
 
-- To run the site locally please go navigate to mini-docs and run - `npm run dev`
-- To add / update docs please go to src
-- To update / add menu items go to src/.vuepress/config.js
+Tutorials and samples for minimal APIs.
 
+- To run the site locally, navigate to mini-docs and run - `npm run dev`.
```

Comment:
where is this?

Reply:
Will update the instructions it should be  minimal.github.io